azurelinuxagent/common/utils/cryptutil.py

```diff
@@ -70,6 +74,15 @@ def decrypt_p7m(self, p7m_file, trans_prv_file, trans_cert_file, pem_file):
         rc = shellutil.run(cmd)
         if rc != 0:
             logger.error("Failed to decrypt {0}".format(p7m_file))
+    
+    def decrypt_encrypted_file(self, private_key, encryptedFile):
```

Comment:
snake case for params, consider `decrypt_file` for naming

Reply:
Fixed

---
In reply to: [192193160](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r192193160) [](ancestors = 192193160)